compiler/src/java_plugin/cpp/java_generator.cpp

```diff
@@ -776,7 +776,7 @@ void GenerateService(const ServiceDescriptor* service,
   vars["MethodType"] = "io.grpc.MethodDescriptor.MethodType";
   vars["ServerMethodDefinition"] =
       "io.grpc.ServerMethodDefinition";
-  vars["BindableService"] = "io.grpc.stub.BindableService";
```

Comment:
Not sure how I feel about moving into io.grpc proper.  The main use case for BindableService is with stubs right?  The point is that it saves you writing it yourself correct?  That sounds like what stubs are for.